backend/app/notes.py

```diff
@@ -0,0 +1,544 @@
+# from https://pages.mtu.edu/~suits/notefreqs.html
```

Comment:
I think we might ultimately cut this part of the visualization, and indeed the feature in the time series app that quantizes the inputs to midi notes, since we're more generally interested in taking input data and mapping it to outputs that don't reference conventional scales, organization of pitches, etc.